recite/admin.py

```diff
@@ -26,8 +26,10 @@ def has_delete_permission(self, request, obj=None):
 
 
 class ReciterForm(forms.ModelForm):
-    """Reciter Form has additional csv_file field of Recitations model"""
-    csv_file = forms.FileField()
+    """Reciter Form has additional segments_file field of Recitations model"""
+    segments_file = forms.FileField()
+    audio_files = forms.FileField(
```

Comment:
I think, we decided that it should be uploaded in the form of zip archive, right?

Reply:
Just saw your comment. I already updated PR with some fixes for also deleting files as well, to finish the Model part so far.